### PR TITLE
Update performance-optimization.md

### PR DESCRIPTION
**Pull request by seneca**
Minor tweaks

### CODE REVIEW

File: packages/nuejs.org/docs/performance-optimization.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 
 # Performance optimization
-Nue gives you performance figures of a text-only website without compromising on design. That is: "as fast as you can get".
+Nue allows you to reach performance of a text-only website without compromising on design, in other words "as fast as you can get".
 
 ## Minimalism
-Lean means fast. Develop your site with [Nue CSS best practices](css-best-practices.html) to get the cleanest and smallest codebase possible with least amount of bloat. It's quite easy to accomplish significant differences on what your codebase looks like:
+Lean, means, fast. Develop your site with [Nue CSS best practices](css-best-practices.html) to get the cleanest and smallest codebase possible with least amount of bloat. It's quite easy to achieve significant reduction in the amount of code that needs to be written:
```
Comment (nobkd) [inline packages/nuejs.org/docs/performance-optimization.md]:
```suggestion
Lean, means fast. Develop your site with [Nue CSS best practices](css-best-practices.html) to get the cleanest and smallest codebase possible with the least amount of bloat. It's quite easy to achieve significant reduction in the amount of code that needs to be written:
```

Really not sure with the commas in the first sentence...

File: packages/nuejs.org/docs/performance-optimization.md [hunk 2]
```diff
@@ -30,15 +30,15 @@ Setting this globally is also possible, but then you need to re-compile all page
 
 ## View transitions
 The next most important performance setting is to enable view transitions.
-This causes your pages to load significantly faster, and the page is transitioned to another in an animated way.
+This causes your pages to load significantly faster, and the page transition in an animated way.
```
Comment (nobkd) [inline packages/nuejs.org/docs/performance-optimization.md]:
```suggestion
This causes your pages to load significantly faster and transition in an animated way.
```

Maybe `and to transition`?

**Review by nobkd — CHANGES_REQUESTED** [review]

File: packages/nuejs.org/docs/performance-optimization.md (hunk 1 above)
Comment (seneca) [reply packages/nuejs.org/docs/performance-optimization.md]: I was wrong. I read it too quickly as Lean, mean, fast. The second comma shouldn't be there as per your changes. And adding "the" is great too.

File: packages/nuejs.org/docs/performance-optimization.md (hunk 2 above)
Comment (seneca) [reply packages/nuejs.org/docs/performance-optimization.md]: That reads much better and no need of comma. Thanks.

**Review by nobkd — APPROVED** [review]

**Comment by nobkd** [comment]
Please check if merge commit is good. Thanks

**Pull request merged**